[20:12:09] /opt/dgl/src/runtime/tensordispatch.cc:43: TensorDispatcher: dlopen failed: /public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/dgl/tensoradapter/pytorch/libtensoradapter_pytorch_1.10.2.so: cannot open shared object file: No such file or directory
Using backend: pytorch
2022/03/27 08:12:12: before drop huge graphs, there are 1178 graphs in DD
2022/03/27 08:12:12: after drop huge graphs, there are 1126 graphs in DD
2022/03/27 08:12:12: ready to generate node embeddings
/public1/home/scfa1517/HKS-Wasserstein-kernel/signature.py:135: ComplexWarning: Casting complex values to real discards the imaginary part
  hks[i] = np.array([np.sum(np.exp(-eigenvalues*t)*eigenvectors[i]*eigenvectors[i]) for t in sample_points])
2022/03/27 08:50:26: the node embedding have been generated
2022/03/27 08:50:27: ready to compute the wass diss with sinkhorn = True and w = 0.30000000000000004
2022/03/27 09:21:47: have computed the wass diss with sinkhorn = True and w = 0.30000000000000004
2022/03/27 09:21:47: ready to compute the wass diss with sinkhorn = True and w = 0.4
2022/03/27 09:52:27: have computed the wass diss with sinkhorn = True and w = 0.4
2022/03/27 09:52:27: ready to compute the wass diss with sinkhorn = True and w = 0.5
2022/03/27 10:23:47: have computed the wass diss with sinkhorn = True and w = 0.5
2022/03/27 10:23:48: ready to compute the wass diss with sinkhorn = True and w = 0.6000000000000001
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:517: UserWarning: Sinkhorn did not converge. You might want to increase the number of iterations `numItermax` or the regularization parameter `reg`.
  warnings.warn("Sinkhorn did not converge. You might want to "
2022/03/27 10:54:58: have computed the wass diss with sinkhorn = True and w = 0.6000000000000001
2022/03/27 10:54:59: ready to compute the wass diss with sinkhorn = True and w = 0.7000000000000001
2022/03/27 11:25:59: have computed the wass diss with sinkhorn = True and w = 0.7000000000000001
[23:26:19] /opt/dgl/src/runtime/tensordispatch.cc:43: TensorDispatcher: dlopen failed: /public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/dgl/tensoradapter/pytorch/libtensoradapter_pytorch_1.10.2.so: cannot open shared object file: No such file or directory
Using backend: pytorch
2022/03/27 11:26:23: before drop huge graphs, there are 1178 graphs in DD
2022/03/27 11:26:23: after drop huge graphs, there are 1126 graphs in DD
2022/03/27 11:26:23: ready to generate node embeddings
/public1/home/scfa1517/HKS-Wasserstein-kernel/signature.py:99: ComplexWarning: Casting complex values to real discards the imaginary part
  wks[i] = np.array([np.sum(np.exp(-(e-log_eigenvalue)*(e-log_eigenvalue)/(2*sigma*sigma))*eigenvectors[i]*eigenvectors[i])/np.sum(np.exp(-(e-log_eigenvalue)*(e-log_eigenvalue)/(2*sigma*sigma))) for e in sample_points])
slurmstepd: error: *** JOB 1120173 ON ia0314 CANCELLED AT 2022-03-28T00:15:28 ***
